src/Mod/Draft/draftguitools/gui_groups.py

```diff
@@ -97,19 +103,8 @@ def Activated(self):
         self.ui.sourceCmd = self
         self.ui.popupMenu(self.labels)
 
-    def proceed(self, labelname):
-        """Place the selected objects in the chosen group or ungroup them.
-
-        Parameters
-        ----------
-        labelname: str
-            The passed string with the name of the group.
-            It puts the selected objects inside this group.
-        """
-        # Deactivate the source command of the `DraftToolBar` class
-        # so that it doesn't do more with this command.
-        self.ui.sourceCmd = None
 
+    def proceed(self, labelname):
```

Comment:
Please don't delete the docstring that I wrote for `proceed`. We want all our methods and functions to have their docstrings.

Reply:
```suggestion
    def proceed(self, labelname):
        """Place the selected objects in the chosen group or ungroup them.
        Parameters
        ----------
        labelname: str
            The passed string with the name of the group.
            It puts the selected objects inside this group.
        """
```